REQUIRE

```diff
@@ -2,3 +2,7 @@ julia 0.5-
 
 Reexport
 LearnBase
+LossFunctions
+PenaltyFunctions
+Transformations
```

Comment:
Is `Transformations` in METADATA? I doubt it. We may need to put a `Pkg.clone` in `.travis.yml` here https://github.com/JuliaML/ObjectiveFunctions.jl/blob/master/.travis.yml#L14 instead